internal/rbd/rbd_healer.go

```diff
@@ -75,12 +72,12 @@ func getSecret(ns, name string) (map[string]string, error) {
 // formatStagingTargetPath returns the path where the volume is expected to be
 // mounted (or the block-device is attached/mapped). Different Kubernetes
 // version use different paths.
-func formatStagingTargetPath(c *k8s.Clientset, pv *v1.PersistentVolume, stagingPath string) (string, error) {
+func formatStagingTargetPath(pv *v1.PersistentVolume, stagingPath string) (string, error) {
 	// Kubernetes 1.24+ uses a hash of the volume-id in the path name
 	unique := sha256.Sum256([]byte(pv.Spec.CSI.VolumeHandle))
 	targetPath := filepath.Join(stagingPath, pv.Spec.CSI.Driver, hex.EncodeToString(unique[:]), "globalmount")
 
-	major, minor, err := kubeclient.GetServerVersion(c)
+	major, minor, err := k8s.GetServerVersion()
```

Comment:
This isn't even needed anymore. The check is for a Kubernetes version that we don't support since a while.